src/GUniLPOracle.t.sol

```diff
@@ -575,7 +575,7 @@ contract GUniLPOracleTest is DSTest {
 
     // --- Fuzz ---
     uint256 constant MAX_PRICE = 1e12 * WAD;   // Max underlying asset Oracle price supported is $1 Trillion USD
-    uint256 constant MIN_PRICE = 10**16;       // $0.01 USD
+    uint256 constant MIN_PRICE = 10**14;       // $0.0001 USD
```

Comment:
Is there any issue with setting this to just 1? It will make things safer.

Reply:
It will fail, `p0 = 1` could be too low and lead to a zero result out of the division.
```haskell
[FAIL] test_sqrt_price_ratio_fuzz(uint256,uint256,uint256,uint256). Counterexample: (25, 115792089237316195423570985008687907853269984665640564039457584007913129639911, 115792089237316195423570985008687907853269984665640564039457584007913129639926, 103372062113308038904937584018315321423557595765614909076903920192225877186311)
```

Reply:
Done in 0438394 (to be used in the overflow fuzz test).